### ISSUE TITLE
error during redisplay for org-sticky-header

### ISSUE THREAD

**Issue by CSRaghunandan** [alphapapa/org-sticky-header#19]
I get the following error in `*Messages*` buffer as I work on a org-file:

```
Error during redisplay: (eval (progn (setq org-sticky-header-stickyline (propertize (org-sticky-header--fetch-stickyline) 'keymap org-sticky-header-keymap)) (list (propertize " " 'display '((space :align-to 0))) 'org-sticky-header-stickyline))) signaled (wrong-type-argument stringp nil) [19 times]
Mark set
```

Not sure why I'm getting this error. Any idea how i can fix this?

**Comment by alphapapa** [comment]
You would have to provide much more information, e.g. a backtrace, software versions, etc.  But first, try to reproduce with `emacs -q`.  Or, more easily, use https://github.com/alphapapa/emacs-sandbox.sh, like:

`emacs-sandbox.sh --install org-sticky-header`

If you can't reproduce the error like that, you will know it's a problem with your config.

**Comment by CSRaghunandan** [comment]
> You would have to provide much more information, e.g. a backtrace, software versions, etc. But first, try to reproduce with `emacs -q`. Or, more easily, use https://github.com/alphapapa/emacs-sandbox.sh, like:
> 
> `emacs-sandbox.sh --install org-sticky-header`
> 
> If you can't reproduce the error like that, you will know it's a problem with your config.

I'm not getting any errors if I enable `toggle-debug-on-error` in emacs. I only see these messages in emacs *Messages* buffer.

Below is my emacs version:
```
Emacs version: GNU Emacs 28.0.50 (build 2, x86_64-pc-linux-gnu, GTK+ Version 3.24.13, cairo version 1.17.3)
 of 2020-01-23, built using commit 5715eb94e90b33ace59dd4c4ccb6e2122bc6db72.

./configure options:
  --with-modules --with-rsvg --with-dbus --with-imagemagick --without-pop --with-xft --with-x-toolkit=gtk3 --with-libotf --with-xml2

Features:
  XPM JPEG TIFF GIF PNG RSVG CAIRO IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY INOTIFY ACL GNUTLS LIBXML2 FREETYPE HARFBUZZ LIBOTF ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS LIBSYSTEMD JSON PDUMPER LCMS2 GMP
```

I am able to reproduce the same error in `emacs -q` by opening any org file.

Version of `org-sticky-header`:     Version: 20191117.549
     Commit: 1053ebdeb3bd14fc8d4538643532efb86d18b73c

This is my configuration for `org-sticky-header`:

```
org-sticky-headers
;; https://github.com/alphaapapa/org-sticky-header
(use-package org-sticky-header
  :hook ((org-mode . org-sticky-header-mode))
  :config
  ;; show full path from the org-mode header
  (setq org-sticky-header-full-path 'full)
  (setq org-sticky-header-always-show-header
        (if org-sticky-header-full-path t nil)))

```

**Comment by alphapapa** [comment]
So you're using a snapshot version of Emacs master.

**Comment by CSRaghunandan** [comment]
> So you're using a snapshot version of Emacs master.

Correct, maybe there are some changes to the display layer that caused this error?

**Comment by alphapapa** [comment]
You'll have to tell me.  :)

**Comment by CSRaghunandan** [comment]
Okay, I updated the emacs snapshot and looks like the error is gone. Maybe it had to do something with emacs display engine changes. But looks like the issue  is fixed.

**Comment by coffeepenbit** [comment]
For anyone else running into this issue, this is what fixed it for me:

I had set `org-sticky-header-always-show-header` to `nil`. Setting it back to the default made the issue go away.

**Comment by alphapapa** [comment]
This appears to be the issue fixed in #20.
